README.md

```diff
@@ -1,11 +1,28 @@
 
-# api-gatekeeper-frontend
+# API Gatekeeper Frontend
 
 [![Build Status](https://travis-ci.org/hmrc/api-gatekeeper-frontend.svg?branch=master)](https://travis-ci.org/hmrc/api-gatekeeper-frontend) [ ![Download](https://api.bintray.com/packages/hmrc/releases/api-gatekeeper-frontend/images/download.svg) ](https://bintray.com/hmrc/releases/api-gatekeeper-frontend/_latestVersion)
 
-This is a placeholder README.md for a new repository
+This is a Play Framework frontend application for HMRC's internal users.
```

Comment:
The API Platform/Programme is a public concept - so perhaps we should include a reference to it here?  Other Readmes have chosen to have a section explaining their tax-specific jargon, in case it isn't clear. 

Maybe something like: 

> This service provides a frontend HMRC's internal users to vet requests to create applications as part of our [HMRC third party tax software and application programming interface (API) strategy](https://www.gov.uk/government/publications/hmrc-third-party-tax-software-and-api-strategy/hmrc-third-party-tax-software-and-application-programming-interface-api-strategy).